.github/CHANGELOG.md

```diff
@@ -120,9 +126,7 @@ Aroosa Ijaz
 
 This release contains contributions from:
 
-Shahnawaz Ahmed, riveSunder, Aroosa Ijaz, Josh Izaac, Nathan Killoran, Maria Schuld.
-
-
+Shahnawaz Ahmed, riveSunder, Aroosa Ijaz, Josh Izaac, Nathan Killoran, Maria Schuld, Johannes Jakob Meyer.
```

Comment:
Did any of your commits make it into v0.4?

Reply:
Sorry, that was an artifact from the merge of the observable_name_refactor branch. Corrected it.